### PR TITLE
Add a new middleware called TransactionControl

### PR DESCRIPTION
**Pull request by fabriziomello**
Add a new middleware called TransactionControl that will be called before the HandleSet in BaseStack.

### CODE REVIEW

**Review by felipeweb — CHANGES_REQUESTED** [review]
It just call next middleware?

**Comment by fabriziomello** [comment]
@felipeweb for now yes, just as I said on issue description prest/prest#265 I've just implement the initial design to check if it's ok and move forward with the main logic of TransactionControl in pREST

**Review by felipeweb — APPROVED** [review]

File: middlewares.go [hunk 1]
```diff
@@ -25,6 +25,13 @@ func HandlerSet() negroni.Handler {
 	})
 }
 
+// TransactionControl is a middleware to handle transactions in pREST
+func TransactionControl() negroni.Handler {
```
Comment (avelino) [inline middlewares.go:29]: I don't understand what this function is doing, call `next`? Should she exist?

Comment (marioidival) [reply middlewares.go:29]: This function do nothing.. Just call next.

Comment (felipeweb) [reply middlewares.go:29]: by what I understood the @fabriziomello  wanted to understand if he is in the right way and now will implement transaction

**Comment by avelino** [comment]
Why is this PR stopped? Needs improvement

File: middlewares.go (hunk 1 above)
Comment (crgimenes) [reply middlewares.go:29]: I believe we can close this PR. The function does nothing, it just calls next and we already have a year that is open. Comments? @fabriziomello @avelino @marioidival @felipeweb

**Comment by crgimenes** [comment]
I closed this PR because it is stopped a long time, if necessary just open again.

**Pull request closed without merging**